develer-staff/webinar_nanabianca
# Add load save and reset functionality

Resolve #1 

❓ What
Add save, using the `s` key, to a file XXX_grid (where XXX is a random generated prefix) the current state of the grid
Add load, launching the program with -f/--filename <path to file> a preexisting grid
Add reset using the `r` key to restore the initial state of the grid 

🔨 Test
- Launch the program and save a couple of grids during the execution, verify the files are correct (matrix of 0/1 with the right size)
- Try launching the program with `python main -f <path to a previously saved grid> and verify it works fine
- Try resetting the grid after a while
